cfme/tests/configure/test_access_control.py

```diff
@@ -1590,7 +1589,24 @@ def test_tenant_unique_automation_domain_name_on_parent_level():
         initialEstimate: 1/2h
         startsin: 5.5
     """
-    pass
+    domain_name = fauxfactory.gen_alphanumeric()
+    domain1 = appliance.collections.domains.create(name=domain_name, enabled=True)
```

Comment:
Optional: Since the creation of the domain is outside of the test scope, why not put this in a fixture?

Reply:
@john-dupuy Agree with you, we can put it in fixture if it is getting used multiple time in same module in different tests and able to re-use same domain obj multiple time, however as it is used one time only and we already using domain collection , so i am not sure if replacing this single line of code with a new fixture is making any time/space optimization. instead it will make a little overhead to jump to the fixture function and then execute the domain collection. Please correct me if i am wrong.